### ISSUE TITLE
Issues with react 17.0.1 and iframes

### ISSUE THREAD

**Issue by jsoutherland** [brammitch/recharts-to-png#6]
This is possibly the same as #5 but the provided solution isn't working in iframes.

You can reproduce the issue with the example at:
https://codesandbox.io/s/recharts-to-png-dyy8q?from-embed

by updating dependencies to:
-   react-scripts: 4.0.1
-   react: 17.0.1
-   react-dom": 17.0.1

That will get you  to the original error as in #5

If you then modify the ``handleDownload`` implementation based on the solution in #5 to:
```javascript
  const handleDownload = React.useCallback(async () => {
    // Send the chart to getPngData
    if (chart && chart.container) {
      const svg = chart.container.children[0];
      const pngData = await getPngData(svg);

      // Use FileSaver to download the PNG
      FileSaver.saveAs(pngData, "test.png");
    }
  }, [chart]);
```

a new error shows up: ``Unable to find element in cloned iframe``.  

These two errors match the behavior in my app under react 17.0.1.

Any thoughts?

**Comment by brammitch** [comment]
Thanks for the great feedback. I released 0.8.0 which includes a hook, `useRechartToPng`, that should work with React 17.

[CodeSandbox](https://codesandbox.io/s/floral-hooks-5v1kl?file=/src/App.tsx)

Let me know if you still have any issues.

**Comment by jsoutherland** [comment]
Thank you for the quick fix!
